components/common-data-items.js

```diff
@@ -509,81 +509,80 @@ OntologyTermDataItems.commonProperties = [
 /**
  * Display data items common to all file-derived objects.
  */
-export function FileDataItems({
-  item,
-  fileSet = null,
-  derivedFrom = null,
-  children,
-}) {
+export function FileDataItems({ item, fileSet = null, children }) {
   return (
     <>
-      {fileSet && (
-        <>
-          <DataItemLabel>File Set</DataItemLabel>
-          <DataItemValue>
-            <Link
-              href={fileSet["@id"]}
-              aria-label={`FileSet ${fileSet.accession}`}
-              key={fileSet.uuid}
-            >
-              {fileSet.accession}
-            </Link>
-          </DataItemValue>
-        </>
-      )}
-      <DataItemLabel>File Format</DataItemLabel>
-      <DataItemValue>{item.file_format}</DataItemValue>
-      <DataItemLabel>Content Type</DataItemLabel>
-      <DataItemValue>{item.content_type}</DataItemValue>
-      <DataItemLabel>md5sum</DataItemLabel>
-      <DataItemValue>{item.md5sum}</DataItemValue>
-      {item.content_md5sum && (
-        <>
-          <DataItemLabel>Content MD5sum</DataItemLabel>
-          <DataItemValue>{item.content_md5sum}</DataItemValue>
-        </>
-      )}
-      {truthyOrZero(item.file_size) && (
-        <>
-          <DataItemLabel>File Size</DataItemLabel>
-          <DataItemValue>{item.file_size}</DataItemValue>
-        </>
-      )}
-      {derivedFrom?.length > 0 && (
-        <>
-          <DataItemLabel>Derived From</DataItemLabel>
-          <DataItemValue>
-            <SeparatedList>
-              {derivedFrom.map((file) => (
+      <DataPanel>
+        <DataArea>
```

Comment:
I wouldn’t put `<DataPanel>` and `<DataArea>` in here because that’s inconsistent with the other common data components here, and people down the line might get confused why this one works differently. It looks a little dumb to then have:
```js
<DataPanel>
  <DataArea>
    <FileDataItems …>
  </DataArea>
</DataPanel>
```
But I think it will be easier to maintain this way.

Reply:
Changed.